pages/_app.js

```diff
@@ -17,7 +17,7 @@ export default class extends App {
     //if the authtoken is not found
     if (typeof c.authtoken == "undefined") {
       //don't do anything if we are on a page that doesn't require credentials
-      if (ctx.pathname == "/login" || ctx.pathname == "/forgot-password")
+      if (ctx.pathname == "/login" || ctx.pathname == "/forgot-password" || ctx.pathname == "/reset-password")
```

Comment:
try to do if (ctx.pathname !== "/") instead and see if it works

Reply:
Is this what we would want? We'd be blacklisting the root endpoint instead of whitelisting specific endpoints that should be done without auth.